### PR TITLE
Fix issue with services loop in wait-healthy

### PR DESCRIPTION
**Pull request by nlisgo**
Currently, we are only checking the status of the first docker container. This ensures that all container ID's are passed to `./scripts/docker/wait-healthy.sh`.

~~Related and blocked by: https://github.com/libero/scripts/pull/15~~

If we can easily filter to only pass containers with healthchecks in place to `./scripts/docker/wait-healthy.sh` then we can close: https://github.com/libero/scripts/pull/15

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -39,7 +39,7 @@ wait-healthy: ## Wait for the containers to be healthy
 		echo "No containers running"; \
 		exit 1; \
 	fi; \
-	for service in $${services}; do \
+	for service in $${services[@]}; do \
```
Comment (giorgiosironi) [inline Makefile:42]: bugfix seems correct from quick experimentation, arcane Make syntax

**Comment by giorgiosironi** [comment]
We should always have health checks in place to understand quickly if a container has a problem. Added one to the `console` service which was making the build fail.

**Comment by thewilkybarkid** [comment]
We should add the healthcheck to the console image itself. /cc @tpluscode

**Comment by giorgiosironi** [comment]
No reason not to push it upstream. Once it is in a release we can remove `healthcheck` from here and it will seamlessly pick up the `HEALTHCHECK` in the image.

**Comment by giorgiosironi** [comment]
https://github.com/hypermedia-app/generic.hypermedia.app/pull/50

File: Makefile (hunk 1 above)
Comment (thewilkybarkid) [reply Makefile:42]: Never seen it before, but https://stackabuse.com/array-loops-in-bash/#loopingoverarrays suggests it's Bash?

**Review by thewilkybarkid — APPROVED** [review]

**Pull request merged**